.docker/images/govcms8/scripts/govcms-deploy

```diff
@@ -68,5 +62,6 @@ fi
 
 # Cache rebuild after distribution update
 if drush status --fields=bootstrap | grep -q "Successful"; then
-  drush cr
+  # drush updb implicitly clears caches after the run.
```

Comment:
Does it do it if there are no db updates:

```
 +++ oc -n sitename-master exec cli-42-pcwpt -i -- sh -c govcms-deploy
 [success] No pending updates.
 [success] Cache rebuild complete.  <---- this is the one in L18
 [success] Database dump saved
```

Reply:
_googles the internet_
https://github.com/drush-ops/drush/blob/master/src/Commands/core/UpdateDBCommands.php#L35
_stands corrected_

Reply:
Yeah :) I did the same, I read through 8.1.5 and 9.x branches they both have the same behaviour.